assets/templates/partials/header.tmpl

```diff
@@ -57,191 +59,251 @@
-                        <li class="primary-nav__child-item  js-expandable__child">
-                            <a class="primary-nav__child-link" tabindex="-1" href="/peoplepopulationandcommunity/healthandsocialcare">{{ localise "HealthSocialCare" .Language 1 }}</a>
-                        </li>
-                        <li class="primary-nav__child-item  js-expandable__child">
-                            <a class="primary-nav__child-link" tabindex="-1" href="/peoplepopulationandcommunity/householdcharacteristics">{{ localise "HouseholdCharacteristics" .Language 1 }}</a>
+                    {{if not .SearchDisabled}}
+                        <li class="nav--controls__item ">
+                            <a href="#nav-search" id="search-toggle" aria-controls="nav-search" aria-expanded="false" class="nav--controls__search">
+                                <span class="nav--controls__text">{{ localise "Search" .Language 1 }}</span>
+                            </a>
                         </li>
-                        <li class="primary-nav__child-item  js-expandable__child">
-                            <a class="primary-nav__child-link" tabindex="-1" href="/peoplepopulationandcommunity/housing">{{ localise "Housing" .Language 1 }}</a>
-                        </li>
-                        <li class="primary-nav__child-item  js-expandable__child">
-                            <a class="primary-nav__child-link" tabindex="-1" href="/peoplepopulationandcommunity/leisureandtourism">{{ localise "LeisureTourism" .Language 1 }}</a>
-                        </li>
-                        <li class="primary-nav__child-item  js-expandable__child">
-                            <a class="primary-nav__child-link" tabindex="-1" href="/peoplepopulationandcommunity/personalandhouseholdfinances">{{ localise "PersonalHouseholdFinances" .Language 1 }}</a>
-                        </li>
-                        <li class="primary-nav__child-item  js-expandable__child">
-                            <a class="primary-nav__child-link" tabindex="-1" href="/peoplepopulationandcommunity/populationandmigration">{{ localise "PopulationMigration" .Language 1 }}</a>
+                    {{end}}
+                </ul>
+                <ul class="wrapper nav-main--hidden primary-nav__list" id="nav-primary" aria-expanded="false">
+                    <li class="primary-nav__item {{if eq $uri " /"}}primary-nav__item--active{{end}} js-nav hide--sm old-ie--display-block">
+                        <a class="primary-nav__link col col--md-7 col--lg-9" href="/">{{ localise "Home" $language 1 }}</a>
+                    </li>
+                    {{ range $item := .NavigationContent }}
+                        <li class="primary-nav__item js-nav js-expandable ">
+                            <a class="primary-nav__link col col--md-8 col--lg-10" href="{{ $item.Uri }}" aria-expanded="false" aria-label="{{ $item.Label }} {{ localise "SubMenu" $language 1 }}">
```

Comment:
`$language` is used inconsistently in and out of new dot contexts (compare to line 85).

Go Templates offer this `$.` notation e.g. `$.Language` notation to give you access to the top level context, useful inside a `range`